BE/src/main/java/com/codesquad/sidedish06/domain/dto/RequestDetail.java

```diff
@@ -0,0 +1,32 @@
+package com.codesquad.sidedish06.domain.dto;
+
+import com.codesquad.sidedish06.domain.entity.DetailSection;
+import com.codesquad.sidedish06.domain.entity.Price;
+import com.codesquad.sidedish06.domain.entity.ThumbImage;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.List;
+
+@Getter
+@Setter
+public class RequestDetail {
+
+    private String hash;
+
+    private String top_image;
```

Comment:
Java의 Naming Convention에 맞지 않은 것 같습니다.

특별한 의도가 있는게 아니라면 수정이 필요해 보입니다.

나머지 부분들에는 코멘트를 남기지 않을게요

```suggestion
    private String topImage;
```